### PR TITLE
[share-via] JSRS migration VCAP-24612

### PR DESCRIPTION
**Pull request by gabrieloliveira41**

### CODE REVIEW

**Comment by gabrieloliveira41** [comment]
Goals verified after changes (2 out of 2 goals total):
- GetVideo
start
get videos
find me a video
- ShareVia
share via email
share 

All resulted in equal intents and equal IDE simulator results after migration comparing to before the migration

**Comment by gabrieloliveira41** [comment]
there is only one warning left
image
but i looked on the dev center and the `viv.shareVia`  recommended version is exactly this one `1.12.1`
image

**Comment by gabrieloliveira41** [comment]
Hello @kangawoo44 @abapayeViv, could you review my PR, please?

File: library-share-via-example/capsule.bxb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 capsule {
   id (example.shareViaExample)
-  version (1.0.0)
+  version (1.1.0)   // JSRS migration VCAP-24612
   format (3)
   runtime-version (7) {
```
Comment (kangawoo44) [inline library-share-via-example/capsule.bxb:5]: while we wait on the library deprecation decision, let's update this to runtime 9

**Comment by gabrieloliveira41** [comment]
the same warning message is shown in the capsule before the migration
image

**Review by kangawoo44 — APPROVED** [review]
library warnings were there before the JSRS migration so this change can go in first and we'll work on fixing the libraries.

**Comment by gabrieloliveira41** [comment]
Hello @abapayeViv, could you review my PR, please?

**Review by abapayeViv — APPROVED** [review]
👍

**Comment by gabrieloliveira41** [comment]
@abapayeViv, could you merge this PR?

**Pull request merged**